Ethminer new arguments & unified pool protocol selection

papampi opened the pull request:
New ethminer command line arguments

papampi committed:
Update 0miner

papampi committed:
ETHMINER_SCHEMA

New ethminer stratum schema

papampi committed:
Version

papampi committed:
Version

papampi commented:
Anything more needed here?

LuKePicci commented:
```
UPOOL="_POOL"
UPORT="_PORT"
**UPROTO="_POOL_PROTOCOL"**
...
xport=$COIN$UPORT
**xproto=$COIN$UPROTO**
…

**if [[ ${!xproto} == "" ]]; then eval $xproto="stratum+tcp"; fi**
```

papampi committed:
Coin Stratum Schema

papampi committed:
Update 1bash.template

papampi committed:
Add stratum schema

Dont think some miners like claymore, dstm, ... need it, so havent add it to them.

papampi commented:
Didn't saw your comment on using  "proto", definitely better 
Already made changes ... will change em again

papampi committed:
Coin Pool Protocol

papampi committed:
Coin Pool Protocol

papampi commented:
Is this correct:
`if [[ ${!xproto} == "" ]]`
or should be:
`if [ -z "$xproto" ]`

papampi committed:
Update 0miner

LuKePicci commented on 1bash.template:
```diff
@@ -608,8 +610,9 @@ plusCPU_WALLET_FORMAT="$WALLET_ADDRESS_FORMAT"
-ETHMINER_OPTS=""                                 # Ethminer optional arguments. add "--api-port -3333" if you want web info
 CLAYMORE_OPTS=""                                 # Claymore optional arguments. EX: add "-esm 2" for miningpoolhub
+ETHMINER_OPTS=""                                 # Ethminer optional arguments. add "--api-port -3333" if you want web info
+ETHMINER_SCHEMA="stratum+tcp"                    # Use Ethminer new schema; stratum+ssl, stratum+tcp, stratum+tls, stratum+tls12, stratum1+ssl, stratum1+tcp, stratum1+tls, stratum1+tls12, stratum2+ssl, stratum2+tcp, stratum2+tls, stratum2+tls12
```
Ok, just drop this line and move the comment  in the notes of the coin section and this is ready.

papampi committed:
bminer coin protocol

papampi committed:
bminer coin protocol

papampi committed:
Update 1bash.template

papampi commented:
Should be done if I'm not missing anything.

LuKePicci reviewed (dismissed):
Awesome.

papampi commented:
Looks like all is good, made a 20 coins switch to test.
Will merge later if all goes well.

papampi committed:
Update 1bash.template
